src/structures/APIMessage.js

```diff
@@ -184,7 +189,7 @@ class APIMessage {
       embeds,
       username,
       avatar_url: avatarURL,
-      allowed_mentions: this.options.allowedMentions,
+      allowed_mentions,
```

Comment:
Accommodate for above change here:
```suggestion
      allowed_mentions: allowedMentions,
```